apple-pie/styles/style.css

```diff
@@ -0,0 +1,147 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  /* font-family: "Roboto", sans-serif; */
+  font-family: "Open Sans", sans-serif;
```

Comment:
use the family that you imported from google :)

Reply:
Yeah, I wasn´t really happy with Roboto and in the French Soup css was Open Sans, so I used that in both :)